lib/cocoapods-search/command/search.rb

```diff
@@ -22,6 +22,7 @@ def self.options
         options += Platform.all.map do |platform|
           ["--#{platform.name.to_s}",     "Restricts the search to Pods supported on #{Platform.string_name(platform.to_sym)}"]
         end
+        options << ['--no-pager',    'Do not pipe search results into a pager']
```

Comment:
I'm inspired here from git `--no-pager` option. I think this feels more natural, regarding users who need no paging option had probably also needed that in git and used `--no-pager` option.

But I'm also OK with `--no-page` if you have a good reasoning.